substratevm/src/com.oracle.svm.jfr/src/com/oracle/svm/jfr/JfrManager.java

```diff
@@ -193,7 +193,7 @@ private static Long parseDuration(Map<JfrStartArgument, String> args, JfrStartAr
                     return Duration.ofMinutes(time).toNanos();
                 } else if ("h".equals(unit)) {
                     return Duration.ofHours(time).toNanos();
-                } else if ("f".equals(unit)) {
+                } else if ("d".equals(unit)) {
```

Comment:
@jiekang This was supposed to be 'd' for days as HotSpot does, not 'f'. I changed that in a separate commit. Keeping it in the same PR as it's related to the same topic.

Reply:
Nice catch, thanks!